ci_framework/roles/tempest/tasks/configure-tempest.yml

```diff
@@ -27,3 +27,16 @@
   ansible.builtin.template:
     src: clouds.yaml.j2
     dest: "{{ cifmw_tempest_artifacts_basedir }}/clouds.yaml"
+
+# As an example:
+# cifmw_tempest_tempestconf_profile:
+#   overrides:
+#     validation.run_ssh: false
+#     telemetry.alarm_granularity: '60'
```

Comment:
will you remove this now seems redundant since you have a fuller explanation in the readme. If you really want a comment here just point to readme?